contribution-guide.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: ballerina-inner-page
+title: The Ballerina Contribution Guide
+permalink: /contribution-guide/
+---
+
+# The Ballerina Contribution Guide
+
+Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is an open-source project under the Apache license and the work of hundreds of contributors.
```

Comment:
```suggestion
Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is an open-source project under the Apache license, with hundreds of contributors.
```

Reply:
How about saying "It is an open-source project under the Apache license, which is owned by the Ballerina community"?

Reply:
Changed to "It is an open-source project under the Apache license, which is owned by the Ballerina community".

The focus should be to say that it is a collaboration of the whole community than mentioning the number of contributors.

Reply:
Not sure if "owned" is what we want to talk about here. Also, to me it reads as though the license is owned by the Ballerina community.

Reply:
Changed as follows;

"Ballerina is an open source programming language and platform for cloud-era application programmers to easily write software. It is licensed under the Apache license and is nurtured by all the contributors of the Ballerina community."